examples/testing-network-shim-app/cypress/support/index.js

```diff
@@ -2,7 +2,7 @@ import { enableNetworkShim } from '@dhis2/cypress-commands'
 import { resetDb } from '../../src/json-server/resetDb'
 
 afterEach(() => {
-    if (Cypress.env('dhis2_api_stub_mode') === 'CAPTURE') {
+    if (Cypress.env('network') === 'capture') {
```

Comment:
I'm wondering if it'd make sense to use constants for these and expose them from the plugin package and use them here (as well as in the commands package)

Reply:
Actually, the helper functions could be exposed instead

Reply:
Done: d1adb66e255fe4330309c369d6d72ec8333b75e1